otter/test/convergence/test_model.py

```diff
@@ -177,3 +189,29 @@ def test_not_done_draining_before_timeout_if_no_connection_info(self):
         node = CLBNode(node_id='1234', description=self.drain_desc,
                        address='10.1.1.1', drained_at=0.0)
         self.assertFalse(node.is_done_draining(now=15, timeout=30))
+
+
+class StepResultTests(SynchronousTestCase):
+    """
+    Tests for :class:`StepResult`.
+    """
+    def test_ordered(self):
+        """
+        Step results are ordered; FAILURE > RETRY > SUCCESS.
```

Comment:
I would hope we don't rely on this ordering. It seems like something that can easily cause confusion -- like, I don't know why they should be in that order and not another one.